dpgen/generator/run.py

```diff
@@ -1310,11 +1310,15 @@ def post_fp_vasp (iter_index,
     system_index.sort()
     
     cwd = os.getcwd()
+
+    tcount=0 
+    icount=0 
     for ss in system_index :
         sys_outcars = glob.glob(os.path.join(work_path, "task.%s.*/OUTCAR"%ss))
         sys_outcars.sort()                
 
         flag=True
+        tcount+=len(sys_outcars)
         for oo in sys_outcars :
             if flag:
```

Comment:
check flat within condition len(_sys)>1 and len(_sys) == 1 ?

Reply:
len(_sys) should be 0 or 1  when  all data collected from static calculation step.